pkg/shell/machines/machines.js

```diff
@@ -53,7 +53,7 @@ export function get_init_superuser_for_options(options) {
      * needs a password and if we don't have one, it will likely fail.
      * That would be okay, but sudo is very noisy about failures and
      * might send nasty emails to your parents.  For that reason we
-     * always pass "init-superuser": "none" here.
+     * pass "init-superuser": "none" here when there is no password.
```

Comment:
Rebase error, this rather belongs in the previous commit. (But if that's the only thing, not a biggie)

Reply:
Ouch, let's fix it anyway.  More test runs, more better.